### ISSUE TITLE
When selecting all Breakpoints, Cant disable/enable BPs

### ISSUE THREAD

**Issue by ScyllaHide** [x64dbg/x64dbg#2925]
### Operating System

Windows 10 Build 19041.1

### x64dbg Version

Jul 20 2022

### Describe the issue

Upon selecting all breapoints in the breakpoint tab, you cant select disable/enable Bps.
If you deselect the head of the "table" you can.

See screenshots

### Steps to reproduce

1. Set bps
2. select them all
3. try to enable/disable them - menu gone? Well deselect "table" head.
![bp_head](https://user-images.githubusercontent.com/12070561/184499512-d2cb2f57-0115-4caa-871e-040a5dfbefa1.png)
![bp_no_head](https://user-images.githubusercontent.com/12070561/184499514-d8be7df6-7d47-4b2e-b44a-9e33cf281933.png)

**Comment by mrexodia** [comment]
Yeah the menus only work for one type, the table head isn’t a breakpoint so you cannot “disable” it. It could be filtered out though, but this seems like a user error to me.

**Comment by Dynabits** [comment]
Can someone please make the "table head" un-selectable, no point having the ability to select it since you can't move it or do anything with it. it just gets in the way!
